scality/metalk8s
# build: fix `stat` command on MacOS

This should fix the issue on MacOS.

How to test:
- If you run `./doit.sh list` once you'll see that it installs a virtualenv first, then you see the output of `doit`.
- If you run `./doit.sh list` you should only see the output of `doit` (the venv is reused)
- If you `touch buildchain/requirements-Darwin.txt` (to fake an update of the dependencies) and then  `./doit.sh list` you should see like the first time (venv is reinstalled and then `doit` output).
